lua/apisix/plugins/jwt-auth.lua

```diff
@@ -75,7 +82,11 @@ function _M.check_schema(conf)
     end
 
     if not conf.secret then
-        conf.secret = core.id.gen_uuid_v4()
+        if base64_secret then
+            conf.secret = ngx_encode_base64(resty_random.bytes(32))
```

Comment:
We should uniformly generate the random number algorithm, or only specified by the user, change the `secret` to a required field

Reply:
changed to use rest_random to generate ascii secret